src/ui/public/doc_table/components/table_row/details.html

```diff
@@ -1,6 +1,20 @@
 <td colspan="{{ columns.length + 2 }}">
-  <a class="pull-right" ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}">
-    <small>Link to /{{row._index}}/{{row._type}}/{{row._id | uriescape}}</small></i>
-  </a>
+  <div class="documentTableRow__actions">
```

Comment:
Ok, that's a simple change. But they already are links and thus allow the user to open them in a new tab. I chose the button styling because the links look so flimsy and tacked-on.